ui/src/containers/NodePageRSP.js

```diff
@@ -123,10 +123,13 @@ const NodePageRSP = (props) => {
 
   const alertList = useAlerts({
     alertname: NODE_ALERTS_GROUP,
-    instance: `${instanceIP}:${PORT_NODE_EXPORTER}`,
   });
 
-  const alertsNode = (alertList && alertList.alerts) || [];
+  const alertsNode = ((alertList && alertList.alerts) || []).filter(
+    (alert) =>
+      alert.labels.instance === `${instanceIP}:${PORT_NODE_EXPORTER}` ||
+      alert.labels.node === name,
+  );
```

Comment:
I am wondering why can't we add this filter within useAlerts() ?
```js
  const alertList = useAlerts({
    alertname: NODE_ALERTS_GROUP,
    instance: `${instanceIP}:${PORT_NODE_EXPORTER}`,
   node: name
  });
  ```

Reply:
That would made an AND filter and then exclude the Alerts with label instance: `${instanceIP}:${PORT_NODE_EXPORTER}` if the node label is not defined.

Reply:
Oh. Yes, that's right.. I miss the OR logic in the filter.. 
Look good to me then!